esx_service/cli/vmdkops_admin.py

```diff
@@ -119,24 +119,48 @@ def commands():
              defined. For example, `tenant create` has a callback, but if a user runs the program
              like: `./vmdkops_admin.py tenant` they will get the following error:
              ```
-             usage: vmdkops_admin.py tenant [-h] {rm,create,set,ls,get} ...
+             usage: vmdkops_admin.py tenant [-h] {rm,create,volume,get} ...
```

Comment:
The comments in L109-L122 still use "vmdkops_admin.py tenant...", which need to be replaced with "vmdkops_admin.py vm-group" since we have renamed "vmdkops_admin.py tenant" command to "vmdk_ops_admin.py vm-group".

In L122, the subcommand for "vmdkops_admin.py vm-group"  are create, update, rm, ls , vm , access. So L122 should be change to something like this:

usage: vmdkops_admin.py vm-group [-h] {create, update, rm, ls, vm, access} ...